### DISCUSSION TITLE
GP Goal Tracker

### DISCUSSION THREAD

**Discussion by SkrillexAkaCraft** [runelite/runelite#15026] [Ideas]
Would be cool to have a GP goal tracker

Ways it could work:

- Unmoon said something about checking how much cash went out of the inventory or how much would end up back in your inventory

- A UI/GUI setting where you can enter how much GP you wanna reach so it tracks your current gp stack then whenever you bought/sold something the value you removed/received will be added/removed to your current cash stack

**Comment by Assadhere** [comment]
Up this!
